docs/Getting_Involved/Newcomers.rst

```diff
@@ -0,0 +1,117 @@
+Welcome to the Newcomers guide!
+===============================
+
+This guide will help you to get involved and teach you how to make your first
+contribution to coala!
+
+Meet the community!
+-------------------
+
+To get started, the first step is to meet the community. We use gitter to
+communicate, and there the helpful community will guide you. Join us here `coala gitter <https://gitter.im/coala-analyzer/coala/>`_.
+The newcomers should ping us "Hello World" to let us know they are here
+because we care!
+
+**Congratulations!** You are now part of our community.
+
+Start working
+-------------
+
+Let us know that you are interested in contributing on gitter and ask for an
+invitation to our org. This is your first step towards contributing.
+The invitation will be sent upon mail and you will have to accept
+it to join. If you don't find the invitation, accept it `here <https://github.com/coala-analyzer>`_.
+
+Now that you are part of our organization, you can start working on issues.
+If you are familiar with git, you can skip the next section and pick an issue.
+
+
+Get help with git
+-----------------
+
+We use github to manage our repository. If you're not familiar with git, we
+strongly recommend following a tutorial, such as `this one <https://try.github.io/levels/1/challenges/1>`_.
+
+If there's anything unclear, or you are encountering problems, feel free
+to contact us on gitter, and we will help you!
+
+Start working
+-------------
+
+Now is the time to pick an issue.
+Here are two links that will lead you to newcomers issues:
+
+- For coala project: `coala newcomer issues <https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`_
+
+- Also, for our coala-bears: `coala-bears newcomer issues <https://github.com/coala-analyzer/coala-bears/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`_
```

Comment:
can you use the generic link I gave you as that inclues newcomer issues for *all* projects?